modules/oe_whitelabel_search/tests/src/Kernel/SearchBlockTest.php

```diff
@@ -123,9 +123,9 @@ public function testBlockRendering(): void {
     $crawler = new Crawler($render->__toString());
 
     // Assert the form rendering.
-    $block = $crawler->filter('#block-whitelabel-search-block');
-    $this->assertCount(1, $block);
-    $form = $block->filter('#oe-whitelabel-search-form');
+    $form = $crawler->filter('#oe-whitelabel-search-form');
+    // The template removes block wrappers for this block.
+    $this->assertSame('body', $form->parents()->getNode(0)->tagName);
```

Comment:
```suggestion
    $form = $crawler->filter('body > #oe-whitelabel-search-form');
    // The template removes block wrappers for this block.
```

Reply:
I like!
But now we have to move the comment elsewhere.